docs_user/modules/proc_adopting-the-loadbalancer-service.adoc

```diff
@@ -2,42 +2,263 @@
+----
+. Certificates
++
+Use the
+`CONTROLLER1_SCP` shell variable to set the value of the existing
+`CONTROLLER1_SSH` variable:
++
+[source,bash]
+----
+$ CONTROLLER1_SCP=$(echo "$CONTROLLER1_SSH" | sed 's/^ssh/scp/g')
 ----
 
-Make sure to replace `<path to the ssh key>` with the correct path to the ssh
-key for connecting to the controller.
+. Run the following set of commands to regenerate the keys and certificates and install the data in {rhocp_long}. These commands convert the existing single certificate authority (CA) configuration into a
+dual CA configuration:
++
+----
+include::../../tests/roles/octavia_adoption/tasks/octavia_certs.yaml[lines="7..83",indent=0]
+----
 
+. (Optional) Public SSH key of Amphorae
++
+These commands will copy the existing public SSH key that can be used for connecting to the amphorae and installs it in Openshift.
++
 [source,bash]
 ----
-include::../../tests/roles/dataplane_adoption/tasks/octavia_certs.yaml[lines="7..83",indent=0]
+include::../../tests/roles/octavia_adoption/tasks/octavia_ssh.yaml[lines="7..20",indent=0]
 ----
 
-These commands convert the existing single CA configuration into a dual CA configuration.
+. Add the {loadbalancer_service} interfaces to each `NodeNetworkConfigurationPolicy` custom resource (CR). The following command adds the network interface to use as the VLAN base interface for the management network for network isolation on the {OpenShiftShort} nodes:
++
+----
+$ oc get -n openstack --no-headers nncp | cut -f 1 -d ' ' | while read; do
 
-// == TODO: Other things
-// TODO
+interfaces=$(oc get nncp $REPLY -o jsonpath="{.spec.desiredState.interfaces[*].name}")
 
-== Enabling the {loadbalancer_service} in OpenShift
-Run the following command in order to enable the {loadbalancer_service} CR.
+(echo $interfaces | grep -w -q "octbr|enp6s0.24") || \
+        oc patch -n openstack nncp $REPLY --type json --patch '
+[{
+    "op": "add",
+    "path": "/spec/desiredState/interfaces/-",
+    "value": {
+      "description": "Octavia VLAN host interface",
+      "name": "enp6s0.24",
+      "state": "up",
+      "type": "vlan",
+      "vlan": {
+        "base-iface": "<enp6s0>", <1>
+        "id": 24
+        }
+    }
+},
+{
+    "op": "add",
+    "path": "/spec/desiredState/interfaces/-",
+    "value": {
+      "description": "Octavia Bridge",
+      "mtu": <mtu>, <2>
+      "state": "up",
+      "type": "linux-bridge",
+      "name": "octbr",
+      "bridge": {
+        "options": { "stp": { "enabled": "false" } },
+        "port": [ { "name": "enp6s0.24" } ]
+        }
+    }
+}]'
 
-[source,bash]
+done
+----
++
+<1> Replace `<enp6s0>` with the name of the network interface in your {OpenShiftShort} setup.
+<2> Replace `<mtu>` with the `mtu` value in your environment.
+
+. Configure the {loadbalancer_service} network attachment definition to connect pods that manage load balancer virtual machines (amphorae) and the OpenvSwitch pods that are managed by the OVN operator:
```

Comment:
Here I'm unsure if this means that you're providing network connectivity to both the pods that manage load balancer virtual machines and the OpenvSwitch pods that are managed by the OVN operator, or if you are more specifically connecting them to each other. So the following suggestion might not be correct.
```suggestion
. Configure the {loadbalancer_service} network attachment definition to connect pods that manage load balancer virtual machines (amphorae) to the OpenvSwitch pods that are managed by the OVN operator:
```

Reply:
could put the goal or outcome first:
"To connect pods that manage load balancer virtual machines (amphorae) and the OpenvSwitch pods that are managed by the OVN operator, configure the {loadbalancer_service} network attachment definition:"

I'm not sure if we have style guidelines on this